### ISSUE TITLE
Improve deployment flexibility

### ISSUE THREAD

**Issue by matthewhanson** [sat-utils/sat-api#68]
Improving the flexibility of the deployment for the user largely depends on #67 . 

We want users to be able to deploy a sat-api and be able to configure:

- the backend (e.g., ES or postgis)
- ingestion endpoints (root STAC catalogs)
- initial ingestion(s) (ECS tasks)
- ongoing ingestion(s) (SNS)

cc @scisco

**Comment by matthewhanson** [comment]
Initial cut at this has happened and on master, closing in favor of future issues.
